test-configs.yaml

```diff
@@ -162,7 +162,7 @@ test_plans:
       - blacklist: *clang_environment_filter
 
   cros-ec:
-    rootfs: debian_buster_ramdisk
+    rootfs: debian_buster-cros-ec_ramdisk
```

Comment:
Please also add a rootfs entry with `debian_buster-cros-ec_ramdisk`, with the URL based on the last staging build i.e. 20200123.3.

Reply:
Pushed a new version using that one from staging

Reply:
There's a rootfs available in production now, see my other comment.